mappers/gige-sdk/rc_genicam_api/find_device.cpp

```diff
@@ -0,0 +1,28 @@
+#include "find_device.h"
+
+extern "C" void find_device() {

```

Comment:
Not clear what's the meaning. Is it to find devices or print?

Reply:
Find GigE Vision devices and list their device information, such as device name, device serial number, etc.